### DISCUSSION TITLE
Falsche Werte beim Rollover

### DISCUSSION THREAD

**Discussion by ujagusch** [jomjol/AI-on-the-edge-device#1959] [Q&A]
Hi zusammen,

bei mir taucht dass Problem auf das beim Rollover (Wechsel des letzten Digits) dieser Rollover nicht sauber erkannt wird.
Beispiel:
Anzeige auf dem Wassermeter - xxx3.999 angezeigt wird  3.999
Anzeige auf dem Wassermeter - xxx3.110 angezeigt wird  3.110 * sollte 4.110 sein
Anzeige auf dem Wassermeter - xxx3.150 angezeigt wird  3.150 * sollte 4.150 sein
Anzeige auf dem Wassermeter - xxx3.200 angezeigt wird  3.200 * sollte 4.200 sein
Anzeige auf dem Wassermeter - xxx4.250 angezeigt wird  4.250

Bei den Werten mit * ist der letzte Digit nicht mehr klar zu lesen, da hier die Anzeigewalze bereits den Rollover duchführt, aber die Zahl nicht erkannt wird und somit der alte Wert beibehalten wird. Erst wenn die neue Zahl sauber zu lesen ist (beendeter Rollover auf dem Wassermeter) wird die neue Zahl auch klar erkannt.

Gibt es eine Möglichkeit der AI mitzuteilen das die neue Zahl bereits dann gültig ist wenn der Analogwert x.11beträgt.

AnalogDigitalTransitionStart scheint mir hier zuständig zu sein, nur komm ich mit der Beschreibung nicht so ganz klar.

Vielleicht könnt mich da jemand in die richtige Richtung schubsen.

Ujagusch

**Comment by caco3** [comment]
schau dir https://jomjol.github.io/AI-on-the-edge-device-docs/Watermeter-specific-analog---digital-transition/ mal an.

**Reply by ujagusch** [reply]
Das Problem - man muss jedesmal warten bis genug Wasser "verbraucht" ist damit es zum Rollover kommt. Aber ich probier mal andere Werte aus.
